packages/components/src/button/index.js

```diff
@@ -86,6 +87,7 @@ export function Button( props, ref ) {
 		describedBy,
 		...additionalProps
 	} = useDeprecatedProps( props );
+	const instanceId = useInstanceId( Button );
```

Comment:
Nit: you can pass the `prefix` in [useInstanceId](https://github.com/WordPress/gutenberg/blob/trunk/packages/compose/src/hooks/use-instance-id/index.js#L34).

Reply:
Thanks and done ✅